lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1306,6 +1310,14 @@ getNetworkInformation (_block0, bp, st) nl = do
     unsafeEpochSucc = fromMaybe bomb . W.epochSucc
       where bomb = error "reached final epoch of the Blockchain!?"
 
+getNetworkParameters
+    :: forall t. ()
+    => (Block, BlockchainParameters, SyncTolerance)
+    -> NetworkLayer IO t Block
+    -> ApiEpochNumber
+    -> Handler ApiNetworkParameters
+getNetworkParameters _ _ _ = error "not implemented"
```

Comment:
I think best practice is to `throwError err501` instead of erroring.

Reply:
agree, fixed

Reply:
ok, but we can use here : 
```
liftIO $ throwIO err501
```

Reply:
I have a memory of `err501` being used in the wallet, but I don't remember exactly how.

With `liftIO $ throwIO err501` we actually get a 500:
```
http http://localhost:8090/v2/network/parameters/latest
HTTP/1.1 500 Internal Server Error
Content-Type: text/plain; charset=utf-8
Date: Thu, 23 Jan 2020 10:08:45 GMT
Server: Warp/3.2.27
Transfer-Encoding: chunked

Something went wrong
```

Which is the same result as `error "not implemeted"`.

Whereas with `Handler $ throwE err501` we at least get a 501 (but with generic code and message)
```
http http://localhost:8090/v2/network/parameters/latest
HTTP/1.1 501 Not Implemented
Content-Type: application/json;charset=utf-8
Date: Thu, 23 Jan 2020 10:10:33 GMT
Server: Warp/3.2.27
Transfer-Encoding: chunked

{
    "code": "unexpected_error",
    "message": "It looks like something unexpected went wrong. Unfortunately I don't yet know how to handle this type of situation. Here's some information about what happened: "
}
```

Reply:
agree. used `Handler $ throwE err501`

Reply:
in the end used Servant's `throwError err501`